contrib/format-iceberg/src/test/resources/lateraljoin/multipleFiles/cust_order_10_1.json

```diff
@@ -0,0 +1,2191 @@
+{
```

Comment:
Please use files present in the java-exec module instead of introducing their copies...